test/views/declaration/summary/SummaryPageViewSpec.scala

```diff
@@ -38,6 +38,8 @@ class SummaryPageViewSpec
     with ViewMatchers {
 
   private val form: Form[LegalDeclaration] = LegalDeclaration.form()
+  private val formWithErrors: Form[LegalDeclaration] =
```

Comment:
was used only once - why not declare it in test case

Reply:
Moved